crates/html-macro/src/parser.rs

```diff
@@ -44,11 +44,11 @@ impl HtmlParser {
     }
 
     pub fn push_tag(&mut self, tag: Tag) {
-        let mut idx = &mut self.current_idx;
-        let mut parent_stack = &mut self.parent_stack;
-        let mut node_order = &mut self.node_order;
-        let mut parent_to_children = &mut self.parent_to_children;
-        let mut tokens = &mut self.tokens;
+        let idx = &mut self.current_idx;
```

Comment:
Thanks a lot for all of this cleanup!

In general the html macro needs some refactoring and commenting but I was a bit spent when I got it working and decided to leave it for another day ...

Reply:
No problem :slightly_smiling_face: